Packs/ThreatGrid/Playbooks/playbook-Detonate_File_-_ThreatGrid.yml

```diff
@@ -484,101 +361,21 @@ inputs:
   value:
     complex:
       root: File
-  required: false
+  required: true
   description: File object of the file to detonate.
-  playbookInputQuery:
-- key: FileName
-  value:
-    simple: file-detonated-via-demisto
-  required: false
-  description: Name of the file to detonate.
-  playbookInputQuery:
-- key: VM
-  value: {}
-  required: false
-  description: The VM to use (string)
-  playbookInputQuery:
-- key: playbook
-  value:
-    simple: default
-  required: false
-  description: Name of the Threat Grid playbook to apply to this sample run
-  playbookInputQuery:
-- key: Private
-  value:
-    simple: "True"
-  required: false
-  description: If the value is set to 'False', the sample will not be private. Any value other than 'False' will set the sample to private.
-  playbookInputQuery:
-- key: Source
-  value: {}
-  required: false
-  description: a string used for identifying the source of the detonation (user defined)
-  playbookInputQuery:
-- key: Tags
-  value: {}
-  required: false
-  description: A comma-separated list of tags applied to this sample.
-  playbookInputQuery:
-- key: Interval
-  value:
-    simple: "1"
-  required: false
-  description: Polling frequency - how often the polling command should run (minutes)
-  playbookInputQuery:
-- key: Timeout
-  value:
-    simple: "15"
-  required: false
-  description: How much time to wait before a timeout occurs (minutes)
-  playbookInputQuery:
+  playbookInputQuery: null
 outputs:
-- contextPath: File.Malicious
-  description: The File malicious description
-  type: unknown
-- contextPath: File.Malicious.Vendor
-  description: For malicious files, the vendor that made the decision
-  type: string
 - contextPath: File.Type
-  description: File type e.g. "PE"
+  description: File Type
```

Comment:
why remove the example?

Reply:
It was changed when we update this playbook (we basically create a new one). I reset the old value.